exdb/tests/integration_tests.py

```diff
@@ -439,13 +444,37 @@ class HallStaffDashboardViewTest(StandardTestCase):
     def test_get_user(self):
         self.create_experience('pe')
         self.create_experience('dr')
-        response = self.clients['ra'].get(reverse('hallstaff_dash'))
+        response = self.clients['ra'].get(reverse('home'))
+
         self.assertEqual(
             response.context["user"].pk,
             self.clients['ra'].user_object.pk,
             "The correct user was not retrieved!"
         )
 
+    def test_number_of_experiences(self):
+        self.create_experience('pe')
+        self.create_experience('dr')
+        response = self.clients['ra'].get(reverse('home'))
+
+        self.assertEqual(len(response.context["experiences"]), 2, "There should be 2 experiences displayed")
+
+    def test_week_ahead(self):
+        self.create_experience('ad')
+        Experience.objects.get_or_create(author=self.clients['ra'].user_object,
+                                         name="E1", description="test description",
+                                         start_datetime=(now() + timedelta(days=2)),
+                                         end_datetime=(now() + timedelta(days=3)),
+                                         type=self.create_type(),
+                                         sub_type=self.create_sub_type(),
+                                         goal="Test Goal",
+                                         audience="b",
+                                         status="ad",
+                                         attendance=3,
```

Comment:
I don't think that it matters in the logic by this point, but if an experience is "upcoming", shouldn't there be no attendance as it has yet to happen?